shared/test/test_files.rb

```diff
@@ -399,19 +399,6 @@ def test_bogus_metadata
     assert not_found?
   end
 
-  def test_metadata_cached
```

Comment:
I just deleted this test entirely; it's 100% redundant with `test_everyone_thumbnail` below.